packages/d/discover/package.yml

```diff
@@ -0,0 +1,47 @@
+name       : discover
+version    : 5.27.10
+release    : 1
+source     :
+    - https://cdn.download.kde.org/stable/plasma/5.27.10/discover-5.27.10.tar.xz : 1f9189e2a759ac035a5ba83a2842a40b7034a7e5885b0836e3921b9671109fa8
+homepage   : https://apps.kde.org/discover/
+license    : GPL-2.0-or-later
+component  : desktop.kde
+summary    : Discover helps you find and install applications, games, and tools. (UNSUPPORTED)
+description: |
+    Warning: this package is unsupported and experimental.
+
+    Discover helps you find and install applications, games, and tools. You can search or browse by category, and look at screenshots and read reviews to help you pick the perfect app. With Discover, you can manage software from multiple sources, including your operating system's software repository, Flatpak repos, the Snap store, or even AppImages from store.kde.org. Finally, Discover also allows you to find, install, and manage add-ons for Plasma and all your favorite KDE apps!
+builddeps  :
+    - pkgconfig(Qt5Quick)
+    - pkgconfig(Qt5Script)
+    - pkgconfig(Qt5WebView)
+    - pkgconfig(flatpak)
+    - pkgconfig(fwupd)
+    - appstream-qt5-devel
+    - attica-devel
+    - extra-cmake-modules
+    - flatpak-devel
+    - karchive-devel
+    - kcmutils-devel
+    - kcrash-devel
+    - kdbusaddons-devel
+    - kdeclarative-devel
+    - kidletime-devel
+    - kio-devel
+    - kirigami2-devel
+    - knewstuff-devel
+    - knotifications-devel
+    - kuserfeedback-devel
+    - packagekit-devel
+    - packagekit-qt-qt5-devel
+    - purpose-devel
+setup      : |
```

Comment:
Currently we build all KDE packages with `optimize: lto`, so that should be added here for consistency. I've also been adding `clang: yes` to all of them for Plasma 6, so you can either add that as well or I'll add it when upgrading this package up to you.